app/views/layouts/application.html.erb

```diff
@@ -21,6 +21,9 @@
   </script>
   <%= javascript_include_tag 'application', 'data-turbolinks-track' => true %>
   <script type="text/javascript" src="<%= extension_javascript_url %>" defer data-turbolinks-track="reload"></script>
+  <script type="text/javascript">
+      moment.locale('<%= Organization.current.locale %>');
```

Comment:
:warning: we already have a variable with locale `mumuki.locale` I think. If we are going to use `moment`, please unify abstractions.